jetty-servlet/src/main/java/org/eclipse/jetty/servlet/FilterHolder.java

```diff
@@ -297,16 +297,8 @@ public String getFilterName()
      * (before their {@link Filter#init(FilterConfig)} method is called)
      * </p>
      */
-    public interface WrapperFunction
-    {
-        /**
-         * Optionally wrap the Filter.
-         *
-         * @param filter the Filter being passed in.
-         * @return the Filter (extend from {@link FilterHolder.Wrapper} if you do wrap the Filter)
```

Comment:
Should probably move this javadoc detail to the holder specific `WrapFunction` javadoc.